test/kubens.bats

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bats
 
 COMMAND="${BATS_TEST_DIRNAME}/../kubens"
+export KUBECTL="$BATS_TEST_DIRNAME/../test/mock-kubectl"
```

Comment:
If we override KUBECTL _only_ when we need the "get namespaces" functionality, would that pretty much be _all_ test cases?

Reply:
it seems that way for now. `get namespaces` is the only call to `kubectl` that requires a cluster